### PR TITLE
 Add Onnx tests and models for L0_infer_variable/zero/rehape

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: qa/L0_infer_variable/test.sh [hunk 1]
```diff
@@ -63,6 +63,17 @@ for TARGET in cpu gpu; do
         done
```
Comment (deadeyegoodwin) [inline qa/L0_infer_variable/test.sh]: Isn't line 62 adding another instance_group for onnx?

File: qa/common/gen_qa_reshape_models.py [hunk 2]
```diff
@@ -493,6 +518,116 @@ def create_ensemble_modelconfig(
...
         input_shapes, input_model_shapes, output_shapes, output_model_shapes)
+
+
+def create_onnx_modelfile(
+        models_dir, model_version, max_batch,
+        dtype, input_shapes, output_shapes):
+
+    # Onnx use string for variable size dimension, and the same string
+    # will be inferred to have same value for the model run.
+    def normalize_variable_shape(shape, increment_index=True):
```
Comment (deadeyegoodwin) [inline qa/common/gen_qa_reshape_models.py]: It seems this function should go in the test_util.py and be called shape_to_onnx_shape to match what we did for TF

File: qa/common/gen_qa_reshape_models.py [hunk 3]
```diff
@@ -493,6 +518,116 @@ def create_ensemble_modelconfig(
...
+
+    io_cnt = len(input_shapes)
+
+    # Use a different model name for the non-batching variant
+    model_name = tu.get_zero_model_name("onnx_nobatch" if max_batch == 0 else "onnx",
+                                   io_cnt, dtype)
+    config_dir = models_dir + "/" + model_name
+    
+    # Must make sure all Onnx models will be loaded to the same GPU if they are
+    # run on GPU. This is due to the current limitation of Onnx Runtime
```
Comment (deadeyegoodwin) [inline qa/common/gen_qa_reshape_models.py]: Can you link to the issue tracking this on onnx site

File: qa/L0_infer_variable/test.sh (hunk 1 above)
Comment (GuanLuo) [reply qa/L0_infer_variable/test.sh]: onnx is not included in the for statement above.

File: qa/common/gen_qa_reshape_models.py (hunk 3 above)
Comment (GuanLuo) [reply qa/common/gen_qa_reshape_models.py]: Create an issue referring to the one on onnx site or just put the link inline?

File: qa/L0_infer_variable/test.sh (hunk 1 above)
Comment (deadeyegoodwin) [reply qa/L0_infer_variable/test.sh]: Can you put  a comment above the for statement calling that out.

File: qa/common/gen_qa_reshape_models.py (hunk 3 above)
Comment (deadeyegoodwin) [reply qa/common/gen_qa_reshape_models.py]: Just inline the link to the existing onnx runtime issue.
